### PR TITLE
#159361807 Improved media gallery

### PR DESCRIPTION
**Pull request by Stylll**
#### What does this PR do?
Update media gallery on product details page

#### Description of Task to be completed?
- add zoom effect to product image on product review page
- update getTopProducts method to filter completed orders
- implement feedback received on the top products analysis design

#### How should this be manually tested?
N/A

#### What are the relevant pivotal tracker stories?
#159361807
#159521098

#### Questions:

### CODE REVIEW

**Review by ozimos — APPROVED** [review]
No issues found. Good work. The magnified view could be larger though.

File: imports/plugins/custom/top-products/client/components/OrderList.js [hunk 1]
```diff
@@ -58,17 +58,21 @@ class OrderList extends React.Component {
     if (Reaction.hasDashboardAccess()) {
       return (
         <div>
-          <div className="table-responsive">
+          <div className="table-responsive text-center">
             <p className="text-center">Filter your result</p>
-            <form>
+            <form className="form-inline">
```
Comment (rovilay) [inline imports/plugins/custom/top-products/client/components/OrderList.js:63]: This doesn't seem to be related to the feature you are currently working.
Can you color more on this?

File: imports/plugins/custom/top-products/client/templates/topProducts.html [hunk 2]
```diff
@@ -1,24 +1,21 @@
 <template name="topProducts">
 
-    <div class="flex flex-column flex-align-center flex-2x comp-order container-lg">
+  <div class="flex flex-column flex-align-center flex-2x comp-order container-lg">
 
-      <div class="flex-item">
-        <div class="panel panel-default user-orders-list">
-          <div class="panel-heading">
+    <div class="flex-item">
```
Comment (rovilay) [inline imports/plugins/custom/top-products/client/templates/topProducts.html:5]: This doesn't seem to be related to the feature you are currently working.
Can you color more on this?

File: package.json [hunk 3]
```diff
@@ -65,6 +65,7 @@
     "later": "^1.2.0",
     "lodash": "^4.17.10",
     "lodash.pick": "^4.4.0",
+    "magnificent": "^2.2.3",
```
Comment (rovilay) [inline package.json]: Nice work so far.
Can I know what this module is used for?, I don't see it being used in any of the files.

**Review by rovilay — COMMENTED** [review]
Great work!
You kept to the team's convention. Keep it up.
Please check out the comments.

File: imports/plugins/custom/top-products/client/components/OrderList.js (hunk 1 above)
Comment (Stylll) [reply imports/plugins/custom/top-products/client/components/OrderList.js:63]: Thanks for noting this. I had to implement the feedback received on the top products analysis feature while working on the improved media gallery feature.

File: imports/plugins/custom/top-products/client/templates/topProducts.html (hunk 2 above)
Comment (Stylll) [reply imports/plugins/custom/top-products/client/templates/topProducts.html:5]: Thanks for noting this. I had to implement the feedback received on the top products analysis feature while working on the improved media gallery feature.

**Review by phemonick — APPROVED** [review]

**Review by rovilay — APPROVED** [review]
